@types/pal/system-info.d.ts

```diff
@@ -48,8 +48,14 @@ declare module 'pal/system-info' {
         public openURL (url: string): void;
         public now (): number;
         public restartJSVM (): void;
-
+        /*
+         * Trigger to exit the game
+         */
         public close (): void;
+        /*
+         * Free the native's resources
+         */
+        public exit(): void;
```

Comment:
What's the difference between close and exit?

Reply:
'close' means close the window
'exit' means really exit the program
![image](https://user-images.githubusercontent.com/5205637/217771563-c933a56c-5571-410e-8de5-a38c22966e52.png)

Reply:
I don't think we should provide two methods to developers. Just one exit method is enough.

Reply:
This interface is internal and is private in game.ts

Reply:
We need an interface to notify native when ts release is complete

Reply:
Ok.